lib/ret/account_context.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Ret.AccountContext do
+  import Canada, only: [can?: 2]
+  alias Ret.{Account}
+
+  def delete_account(%Account{} = acting_account, %Account{} = account_to_delete) do
+    if can?(acting_account, delete_account(account_to_delete)) do
+      case Ret.Repo.delete(account_to_delete) do
+        {:ok, _} -> {:ok}
```

Comment:
The idiom would be to return `:ok` bare. You’d only wrap it in a tuple if you were tagging a value (e.g., `{:ok, 42}`). Monuples (1-tuples) aren’t ever needed in Erlang or Elixir.